src/Equinor.ProCoSys.IPO.Command/EventHandlers/PostSaveEvents/IpoCompletedEmailHandler.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.Domain.Events.PostSave;
+using Equinor.ProCoSys.IPO.Email;
+using MediatR;
+using Microsoft.Extensions.Options;
+
+namespace Equinor.ProCoSys.IPO.Command.EventHandlers.PostSaveEvents
+{
+    public class IpoCompletedEmailHandler : INotificationHandler<IpoCompletedEvent>
```

Comment:
We should discuss how we handle this, as this is the first time we have two different "listeners" for the same event. I think we should have one handler handling the IpoCompletedEvent.  

If email service is down, we currently miss these emails, as they are not stored until handled